src/main/java/mate/academy/spring/model/dto/CinemaHallTitleIdDto.java

```diff
@@ -0,0 +1,27 @@
+package mate.academy.spring.model.dto;
+
+public class CinemaHallTitleIdDto {
```

Comment:
```suggestion
public class CinemaHallResponseDto {
```
Better not to use fields in names. In real model you will have a lot of fields so naming will get not very informative

Reply:
I already have CinemaHallResponseDto. CinemaHallTitleIdDto only contains id and name

Reply:
I agree that the name is not the best.